onnxruntime/core/optimizer/transpose_optimization/ort_transpose_optimization.cc

```diff
@@ -89,6 +89,28 @@ constexpr HandlerInfo max_pool_op_handler = {&FirstInput, &HandleMaxPool};
 constexpr HandlerInfo node_1_inp_handler = {&FirstInput, &HandleSimpleNode};
 constexpr HandlerInfo reduce_op_handler = {&FirstInput, &HandleReduceOps};
 
+static bool HandleContribQuantizeDequantizeLinear(HandlerArgs& args) {
```

Comment:
can we call the base handler instead of redefining? In my draft PR I exposed some of those implementations so they could be re-used.

Reply:
I don't think we can directly use the base handler because it checks the opset version to determine if the QDQ op has an axis attribute. The ms_domain ops always have an axis attribute regardless of the opset.

That said, I am exposing the `TransposeQuantizeDequantizeAxis` utility from onnx_transpose_optimization.h in order to reuse the logic that transposes the axis attribute. This function is called on the line below.